### DISCUSSION TITLE
Vercel CLI Proxy Config

### DISCUSSION THREAD

**Discussion by zernonia** [vercel/vercel#4752] [Help]
I found a question similar to my situation, I'm trying to setup Vercel cli on my company pc, where proxy is required. Is there a command for vercel cli to include the proxy config for every command? 

I'm stuck at login where the request to `https://api.vercel.com/now/registration?mode=login` failed due to ECONNREFUSED. I've tried connecting to the url using browser, it shows some json error (obviously since it does not have any payload)

So I'm guessing it would be the vercel cli proxy setting. I saw git has some config for proxy.. is there similar features in vercel cli? thanks

**Comment by paulogdm** [comment]
Hello,

Unfortunately, we don't support proxies at the moment.

**Reply by kimmy1991** [reply]
Hi,

Do we support this feature yet? We have the same issue and we are in need of the cli supporting a proxy.

thanks
